auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -1177,22 +1285,21 @@ private void consumeContent(final XMLStreamReader reader) throws XMLStreamExcept
             }
         }
 
-        private boolean validNamespace(final String namespace) {
-            return ELYTRON_1_0.equals(namespace) || ELYTRON_1_0_1.equals(namespace);
-        }
     }
 
-    static final class LoadedIdentity {
+    protected static final class LoadedIdentity {
         private final String name;
         private final List<Credential> credentials;
         private final Attributes attributes;
         private final Encoding hashEncoding;
+        private final Supplier<Provider[]> providers;
 
-        LoadedIdentity(final String name, final List<Credential> credentials, final Attributes attributes, final Encoding hashEncoding) {
+        LoadedIdentity(final String name, final List<Credential> credentials, final Attributes attributes, final Encoding hashEncoding, final Supplier<Provider[]> providers) {
             this.name = name;
             this.credentials = credentials;
             this.attributes = attributes;
             this.hashEncoding = hashEncoding;
+            this.providers = providers;
```

Comment:
Just curious, why was this added here? Is it used anywhere?

Reply:
It's not necessary. Removed now